src/Controller/Admin/ActivityController.php

```diff
@@ -190,7 +214,7 @@ public function imageAction(Request $request, Activity $activity): Response
     }
 
     /**
-     * Deletes a ApiKey entity.
+     * Deletes an activity.
      */
     #[Route('/{id}/delete', name: 'delete')]
     public function deleteAction(Request $request, Activity $activity): Response
```

Comment:
Please add a check that only archived activities can be deleted.

Reply:
I would, but as noted below. It was my understanding that deletion fell outside the scope of this PR. (in an attempt to keep our PR's a little smaller and more focused on a single change).

I changes the text because the sloppy copy paste work someone did was starting to bother me. If you prefer I can revert it back to the original(wrong) text.